### ISSUE TITLE
convolution layer, dropout layer and biases

### ISSUE THREAD

**Issue by MarvinYeh** [jakeret/tf_unet#152]
Hi I am working through the code and trying to make it 3D.
I have several questions regarding the original architecture of Unet.
1. In the original paper by Dr. Ronnerberg, it seems that there is no dropout layer after each convolution layer? 
2. The bias was always added after dropout. Is there any difference compare to one that added bias before dropout?
Many thanks for your help.

**Comment by jakeret** [comment]
Sorry for the late reply. Yes you are right. The architecture is slightly different. I never checked if this is making a difference.
Have you had any success in implementing a 3d version?

**Comment by MarvinYeh** [comment]
Thanks for the reply. 

Hopefully I could push my code later this week. 

As you've mentioned in other issue post that simply change some of the vectors should do the work.
This also suggest that all the training data should has the same 'volume', which might be a problem when I work on MRI data. Patients tend to have different slices, hence the depth won't be the same.

**Comment by MarvinYeh** [comment]
I've updated my branch with the 3D unet structure. I also slightly modified the toy generator to make a stack of images for training. 
One thing bothers me. Where is tensorflow imported? I didn't find it and can't get my GPU to accelerate the training.
Any idea?

**Comment by jakeret** [comment]
Tensorflow is imported [here](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L28)

**Comment by yiqianglee** [comment]
@jakeret @MarvinYeh Is there any update for question 2? i just submit a same issue before i see this issue.

/////////////////
The bias was always added after dropout. Is there any difference compare to one that added bias before dropout?
